modules/settings/Settings.js

```diff
@@ -13,8 +13,15 @@ function getLocalStorage() {
 
     // eslint-disable-next-line no-invalid-this
     const global = typeof window === 'undefined' ? this : window;
+    let storage;
 
-    return global.localStorage;
+    try {
+        storage = global.localStorage;
```

Comment:
ok I see, thanks for the explanation. About the variable I don't mind, only if you have time to add it.